docs/install.md

```diff
@@ -104,101 +101,66 @@ Running the script without any arguments prints a description of all commands.
 
     ./bin/fluo
 
-## Configure a Fluo application
-
-You are now ready to configure a Fluo application. Use the command below to create the
-configuration necessary for a new application. Feel free to pick a different name (other than
-`myapp`) for your application:
-
-    fluo new myapp
-
-This command will create a directory for your application at `apps/myapp` of your Fluo install which
-will contain a `conf` and `lib`.
-
-The `apps/myapp/conf` directory contains a copy of the `fluo.properties` from your default
-configuration. This should be configured for your application:
-
-    vim apps/myapp/fluo.properties
-
-When configuring the observer section in fluo.properties, you can configure your instance for the
-[phrasecount] application if you have not created your own application. See the [phrasecount]
-example for instructions. You can also choose not to configure any observers but your workers will
-be idle when started.
-
-The `apps/myapp/lib` directory should contain any observer jars for your application. If you
-configured [fluo.properties] for observers, copy any jars containing these observer classes to this
-directory.
-
-## Initialize your application
-
-After your application has been configured, use the command below to initialize it:
+## Initialize Fluo application
```

Comment:
Related to this, I was thinking the docs should provide a nice conceptual overview of how Fluo stores observer jars in HDFS and app config in ZooKeeper.